### ISSUE TITLE
Prevent re-ordering in home Droppable

### ISSUE THREAD

**Issue by joetidee** [atlassian/react-beautiful-dnd#516]
I have two lists (A and B) of Draggable components. I want to be able to drag from A to B only and I do not want to be able to re-order the items in list A - How do I achieve this?

**Comment by alexreardon** [comment]
Set isDropDisabled to true on DroppableA

**Comment by alexreardon** [comment]
More info: https://github.com/atlassian/react-beautiful-dnd#conditionally-dropping

**Comment by joetidee** [comment]
You legend :)

**Comment by gnapse** [comment]
How about if I allow to reorder, but I want the first element on the list to always be the first element? I already set `isDragDisabled={index === 0}` to prevent users from drag the first element, but nothing is preventing them to drag the second element on to the first position.

**Comment by gnapse** [comment]
I figured it out. I needed to avoid wrapping the first column altogether in a draggable.

The problem is that it became a bit cumbersome, because I had to then render the component in two places, one where I do not want it wrapped by the draggable, and in the else, render a draggable with the same component in it. In the first case ignoring the draggable `provided` stuff.

Of course I managed to make it DRY and all. Great library BTW!
